docs/src/main/paradox/overview.md

```diff
@@ -25,7 +25,7 @@ It features:
 
 * Akka gRPC $project.version$ ([Github](https://github.com/akka/akka-grpc))
 * Scala $scala.binary.version$
-* Akka Http $akka-http.version$ (@extref[Docs](akka-http:index.html), [Github](https://github.com/akka/akka-http))
+* Akka Http $akka-http.version$ (@extref[Docs](akka-http-docs:index.html), [Github](https://github.com/akka/akka-http))
```

Comment:
This change is because we have it like that in Alpakka. I would rather make it shorter in Alpakka instead.

Reply:
It had `-docs` attached in others, as well. Just make it consistent...